### ISSUE TITLE
It has been marked as malicious by windows defender

### ISSUE THREAD

**Issue by noonewhoim** [tanc7/EXOCET-AV-Evasion#4]
Is there any way to solve it?

**Comment by tanc7** [comment]
I will be working on it after my accounting exam on Wednesday. I assume go-memexec got picked up?

**Comment by lu4p** [comment]
Garble should be able to obfuscate a 'single' file, or anything which can be done in a go build command `garble build file.go`.

**Comment by czonta96** [comment]
Yes, stop testing samples on VirusTotal.
https://scanme.org

**Comment by tanc7** [comment]
Hi everyone, I am currently looking through the materials for the OSEP course (Offensive Security Experienced Penetration Tester). I highly recommend czonta96's link. As of right now there are tons of obstacles introduced since Windows 8.1, including but not limited to, AMSI bypasses, Windows Defender's scantime and runtime detection. That, and I am balancing multiple personal issues in my life right now, like getting my house fixed up from the FBI raid, dental appointments (their suggested bill is over $46,000 for my treatment), medical appointments, counseling appointments, and balancing between studying cyber and finance, as well as using what I have learned in my BSBA of Accounting every day.

Until I update the repo, I suggest using a combination of powershell scripts and .NET commands such as Powershell Reflection, patching of running processes, and registry modifications to shut down AMSI (as it works independently of WinDefend.exe). In Windows 10, the Windows Script Host (cscript.exe) can run jscript files (it's a Microsoft version of JavaScript that also can implement .NET Assembly code as well as other MS scripting languages like vbs and wscript) to help you bypass Defender.

Furthermore there is a great link from IppSec [](https://www.youtube.com/watch?v=9pwMCHlNma4) to dissect the binary in question using the `head -c` command to slice up the binary from the beginning to find what is being flagged by signature detection, or the `tail -c` command to slice the bytes from the bottom, have it scanned until you reach specific byte offsets, hex dumping it with `xxd -p binary`, and then manually rewriting Exocet (change the offending characters and function calls from upper to lower and vice-versa, and detect whether or not it's the Exocet decrypter stub or the payload itself that requires modification).

Exocet is a extremely simple app that ingests detectable malware and adds it's own layer of encoding and encryption.

**Comment by czonta96** [comment]
Your house fixed by FBI raid lol
A good VPN is a thing to prevent that annoyers from knocking your door

**Comment by tanc7** [comment]
Yes. But unfortunately commercial VPNs like IPVanish will respond to
federal subpoenas. I read it in court discoveries of a guy in Chicago. Poor
dude bought 200 hacked twitter accounts and thought he could hide behind
IPVanish, then because of that he got superseded by CFAA after being
charged with terrorism.

Best option is to use bulletproof VPNs or even better, make your own VPN
with WireGuard. I wouldn’t trust IPSec.

Anyways I’m through with black hat stuff and just play CTFs now and
periodically writing code, as well as pursuing my second college degree and
managing my family’s brokerage account on Fidelity. My supervised release
conditions forbid me from any job involving computers or handling
personally identifiable information, even as a cashier at Walmart. Planning
to move to Argentina when my 29 months are up.

**Comment by czonta96** [comment]
Is Nord VPN good?

**Comment by czonta96** [comment]
Anyway seems like Defender can't be disabled easily by a meterpreter session becouse when you try to enter in CMD shell mode it rages and shut down your session

**Comment by tanc7** [comment]
I can only answer this question in a manner that it could not incriminate me or get me indicted. I suggest heavy research into “bulletproof hosting” providers, spin up a VPS and convert it into a client-to-site VPN. AlgoVPN does just that, which is a collection of Ansible scripts to quickly configure a secure WireGuard VPN endpoint. 

https://github.com/trailofbits/algo

If you own the data, and have firm control over who logs it, your “mostly” clean. You just have to trust your hosting providers. Some of them offer warrant canaries, usually a PGP key to verify the service they are providing is “solid”. 

On the flip side, some bulletproof hosting providers have flipped to the Feds, I personally know a handful of inmates who confirmed it from the InFraud Indictment in Federal prison in Pahrump, NV.

Just be careful and do clear research.

I even shook R. Kelly’s hand at Chicago MCC in late 2019.
